Resources/views/Registration/confirmed.html.twig

```diff
@@ -5,7 +5,7 @@
 {% block fos_user_content %}
     <p>{{ 'registration.confirmed'|trans({'%username%': user.username}) }}</p>
     {% if app.session is not empty %}
-        {% set targetUrl = app.session.get('_security.' ~ app.security.token.providerKey ~ '.target_path') %}
+        {% set targetUrl = app.session.get('_security.' ~ providerKey ~ '.target_path') %}
```

Comment:
I suggest moving the whole target url retrieval in the controller instead, and passing it to the template. It would be cleaner